lib/system/sysstr.nim

```diff
@@ -259,7 +259,7 @@ proc incrSeqV2(seq: PGenericSeq, elemSize: int): PGenericSeq {.compilerProc.} =
     result.reserved = r
 
 proc setLengthSeq(seq: PGenericSeq, elemSize, newLen: int): PGenericSeq {.
-    compilerRtl.} =
+    compilerRtl, inline.} =
```

Comment:
Use ``inl`` here which is a variant of ``inline`` that knows about DLL boundaries.